react/features/filmstrip/components/native/Filmstrip.js

```diff
@@ -229,13 +235,15 @@ class Filmstrip extends PureComponent<Props> {
             ? width / (thumbnailWidth + (2 * margin))
             : height / (thumbnailHeight + (2 * margin))
         );
-        const participants = this._separateLocalThumbnail ? _participants : [ _localParticipantId, ..._participants ];
+        const participants = this._separateLocalThumbnail || _disableSelfView
```

Comment:
I think we should keep the local thumbnail in filmstrip mode, just not render the video. This also makes re-enabling the self-view easier since the control is just there.